packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -36,6 +38,10 @@ const getTabStyles = stylesFactory((theme: GrafanaTheme) => {
       &:focus {
         color: ${colors.linkHover};
       }
+
+      .counter {
+        margin-left: ${theme.spacing.sm};
```

Comment:
can we add this to the counter component instead now that it's an internal component of the tab, then we do not need the cx() class combining

Reply:
yes, good catch!